src/pages/Authentication/Login.jsx

```diff
@@ -2,14 +2,16 @@ import "./Authentication.css";
 import { useState } from "react";
 import { Link } from "react-router-dom";
 import { useNavigate } from "react-router";
-import { useAuth, useLikes } from "../../hooks";
+import { useAuth, useLikes, useHistory, usePlaylists } from "../../hooks";
```

Comment:
You might have to consider renaming your custom hook `useHistory` to something else because as far as I know from the little react I learnt in the past, `useHistory` hook already exists in declarative routing of react docs. Please refer [https://v5.reactrouter.com/web/api/Hooks](https://v5.reactrouter.com/web/api/Hooks) for more info. Although I am not sure if we can have multiple hooks (custom as well as inbuilt) with the same name or not

Reply:
Great find @RakshaPawar108. I did not know about this, but I did some research, and since we are using react-router v6 we do not need to worry about `useHistory`. Read more about it [here](https://pranshudobhal.hashnode.dev/whats-new-in-react-router-v6#:~:text=The%20old%20useHistory%20hook%20has%20been%20removed%20and%20replaced%20by%20an%20all%2Dnew%20useNavigate%20hook.)

Reply:
Thanks for the blog link here @kart-c , I did use the `useNavigate` hook in my own project but I was not aware that it replaced the `useHistory` hook from react-router v5. Now it is clear to me.

Reply:
I didn't knew about useHistory at all, got to know something new. But as Kartik mentioned that useHistory is depreciated and replaced by useNavigate now, so I guess it's fine now using useHistory. Thanks for reviewing and telling me something knew.